### ISSUE TITLE
Update Spiceserver URL

### ISSUE THREAD

**Issue by Kelvinrr** [DOI-USGS/ISIS3#5327]
**Description**  

We should update the spiceserver URL to default to https://astrogeology.usgs.gov/apis/ale/v0.9.1/spiceserver/ 

This new AWS backed service should be more powerful and generally more useful and give us a better avenue for updating the service over time. 

See the [RFC](https://github.com/DOI-USGS/ISIS3/discussions/5293) for more details. 
 
This change would have to be backported to older versions of ISIS.

**Example**  

Basically update this line: https://github.com/DOI-USGS/ISIS3/blob/dev/isis/src/base/apps/spiceinit/spiceinit.xml#L862

Then issue a deprecation warning for the old URL.

**Comment by amystamile-usgs** [comment]
@Kelvinrr What would be a good message for the deprecation? Something like the url will be going away or may no longer work? Also not sure how to back port this change to older versions of ISIS.

**Comment by rvwagner** [comment]
I see a version number in that URL- does that mean the new URL will at some point be changed due to having a new version?  Is there some coordination needed between ISIS versions and versions in that URL? Will this new URL be supported long-term even if there are new versions in the future? (Including keeping the kernels it has access to up-to-date?)

If it is expected that the default URL will change over time, can you modify spiceinit to allow redirects, or possibly host a page somewhere where you list what URLs are currently available and compatible with what ISIS versions, which spiceinit could direct users to if its default URL is unresponsive?
